docs/source/install.rst

```diff
@@ -112,6 +110,18 @@ Then run:
     $ ../configure_dealii
     $ make -j<N> install
 
+Please refer to the deal.II documentation to see how to install p4est and
+Trilinos.
+
+Here is how you can build p4est:
```

Comment:
I don't think this is necessary. deal has a script to install p4est. Just send them to deal.